.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
-        id: generate-release-notes
```

Comment:
- The output formatting in the create-confluence-page job seems to be inconsistent. Ensure uniformity in how the commit details are displayed.